projects/core/components/scrollbar/scrollbar.component.ts

```diff
@@ -47,7 +53,10 @@ export class TuiScrollbarComponent {
     readonly browserScrollRef = new ElementRef(this.elementRef.nativeElement);
 
     constructor(
-        @Inject(CSS) private readonly cssRef: any,
+        @Inject(CSS_TOKEN)
+        private readonly cssRef: TuiInjectionTokenType<
+            typeof import('@ng-web-apis/common').CSS
```

Comment:
Shouldn't this be `TuiInjectionTokenType<CSS_TOKEN>`?

Reply:
oh, sorry, yes)

Reply:
fixed